apps/src/lab2/types.ts

```diff
@@ -18,6 +18,8 @@ export interface Channel {
   publishedAt: string | null;
   createdAt: string;
   updatedAt: string;
+  // Optional lab-specific configuration for this project
```

Comment:
would it be worth including in the comment here that this will be saved to the projects table?

Reply:
Thanks for catching!  Added some comments.